openquake/hazardlib/gsim/base.py

```diff
@@ -212,6 +220,148 @@ def _get_poes(mean_std, loglevels, truncation_level, squeeze=False):
     return _truncnorm_sf(truncation_level, out)
 
 
+from scipy.stats import norm
```

Comment:
This looks like a duplication of hazardlib.site_amplification.norm_cdf. Why not importing that?

Reply:
I have a circular import problem if I do:
```
from openquake.hazardlib.site_amplification import norm_cdf
```

Reply:
The solution is to move norm_cdf in another module (perhaps in hazardlib.stats, since it is a statistical function?)

Reply:
I have a circular import problem.

Reply:
Done